includes/wikia/nirvana/WikiaParserTagController.class.php

```diff
@@ -0,0 +1,35 @@
+<?php
+abstract class WikiaParserTagController extends WikiaController {
+
+	/**
+	 * @desc Checks if parameter from parameters array is valid
+	 *
+	 * @param String $paramName name of parameter which should get validated
+	 * @param String|Mixed $paramValue value of the parameter
+	 * @param String $errorMessage reference to a string variable which will get error message if one occurs
+	 *
+	 * @return bool
+	 */
+	protected function isTagParamValid( $paramName, $paramValue, &$errorMessage ) {
+		$isValid = false;
+
+		$validator = $this->buildParamValidator( $paramName );
+		if( !is_null($validator) && $validator instanceof WikiaValidator ) {
+			$isValid = $validator->isValid( $paramValue );
+
+			if( !$isValid ) {
+				$errorMessage = $validator->getError()->getMsg();
+			}
+		}
+
+		return $isValid;
+	}
+
+	/**
+	 * @desc Factory method to create validators for params; returns null if validator can't be created
+	 *
+	 * @param String $paramName
+	 * @return null|WikiaValidator
```

Comment:
to me false is more descriptive
it's like program do tell you that something
false is a value

null on the other hand is also falsy
but null is not a value

null is nothing basically, it's just happens to be falsy

to be honest it would make sense if this function throws if it can't create a validator
as that means that someone is trying to create something that cannot exist